src/runtime_src/core/tools/common/tests/TestBandwidthKernel.cpp

```diff
@@ -281,7 +280,7 @@ TestBandwidthKernel::runTest(std::shared_ptr<xrt_core::device> dev, boost::prope
   }
 
   int num_kernel = 0;
-  int num_kernel_ddr = 0;
+  unsigned int num_kernel_ddr = 0;
```

Comment:
Please fix the logic for num_kernel also.

Reply:
hi @chvamshi-xilinx fixed the logic for num_kernel. made it unsigned int.